### ISSUE TITLE
Random crash

### ISSUE THREAD

**Issue by GabclarexxFR** [WaterMediaTeam/watermedia#2]
My game randomly crash when a youtube video is played

I don't get a crash report but a simple "error  code-3" on the launcher

I get this from my lastest.log

````
 [media-events/ERROR]: An error occurred building event handler
java.lang.NullPointerException: Cannot invoke "java.lang.ClassLoader.loadClass(String)" because the return value of "java.lang.Thread.getContextClassLoader()" is null
	at net.minecraftforge.eventbus.EventSubclassTransformer.buildEvents(EventSubclassTransformer.java:92) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventSubclassTransformer.transform(EventSubclassTransformer.java:44) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventBusEngine.processClass(EventBusEngine.java:26) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.service.ModLauncherService.processClassWithFlags(ModLauncherService.java:32) ~[eventbus-6.0.3.jar:6.0.3+6.0.3+master.039e4ea9]
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) [securejarhandler-2.1.4.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) [?:?]
	at me.srrapero720.watermedia.api.video.players.VideoLanPlayer$1.playing(VideoLanPlayer.java:260) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerPlayingEvent.notify(MediaPlayerPlayingEvent.java:36) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerPlayingEvent.notify(MediaPlayerPlayingEvent.java:28) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager.raiseEvent(NativeEventManager.java:179) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager$EventCallback.callback(NativeEventManager.java:206) [watermedia-1.2.0.jar%23366!/:1.2.0]
[21:25:08] [media-events/ERROR]: An error occurred building event handler
java.lang.NullPointerException: Cannot invoke "java.lang.ClassLoader.loadClass(String)" because the return value of "java.lang.Thread.getContextClassLoader()" is null
	at net.minecraftforge.eventbus.EventSubclassTransformer.buildEvents(EventSubclassTransformer.java:92) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventSubclassTransformer.transform(EventSubclassTransformer.java:44) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventBusEngine.processClass(EventBusEngine.java:26) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.service.ModLauncherService.processClassWithFlags(ModLauncherService.java:32) ~[eventbus-6.0.3.jar:6.0.3+6.0.3+master.039e4ea9]
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) [securejarhandler-2.1.4.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) [?:?]
	at me.srrapero720.watermedia.api.video.players.VideoLanPlayer$1.playing(VideoLanPlayer.java:263) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerPlayingEvent.notify(MediaPlayerPlayingEvent.java:36) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerPlayingEvent.notify(MediaPlayerPlayingEvent.java:28) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager.raiseEvent(NativeEventManager.java:179) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager$EventCallback.callback(NativeEventManager.java:206) [watermedia-1.2.0.jar%23366!/:1.2.0]
[21:25:08] [Thread-21/ERROR]: An error occurred building event handler
java.lang.NullPointerException: Cannot invoke "java.lang.ClassLoader.loadClass(String)" because the return value of "java.lang.Thread.getContextClassLoader()" is null
	at net.minecraftforge.eventbus.EventSubclassTransformer.buildEvents(EventSubclassTransformer.java:92) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventSubclassTransformer.transform(EventSubclassTransformer.java:44) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventBusEngine.processClass(EventBusEngine.java:26) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.service.ModLauncherService.processClassWithFlags(ModLauncherService.java:32) ~[eventbus-6.0.3.jar:6.0.3+6.0.3+master.039e4ea9]
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) [securejarhandler-2.1.4.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) [?:?]
	at me.lib720.caprica.vlcj.player.embedded.videosurface.NativeBuffers.allocate(NativeBuffers.java:74) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.embedded.videosurface.CallbackVideoSurface$SetupCallback.format(CallbackVideoSurface.java:93) [watermedia-1.2.0.jar%23366!/:1.2.0]
[21:25:08] [Thread-21/ERROR]: An error occurred building event handler
java.lang.NullPointerException: Cannot invoke "java.lang.ClassLoader.loadClass(String)" because the return value of "java.lang.Thread.getContextClassLoader()" is null
	at net.minecraftforge.eventbus.EventSubclassTransformer.buildEvents(EventSubclassTransformer.java:92) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventSubclassTransformer.transform(EventSubclassTransformer.java:44) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventBusEngine.processClass(EventBusEngine.java:26) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.service.ModLauncherService.processClassWithFlags(ModLauncherService.java:32) ~[eventbus-6.0.3.jar:6.0.3+6.0.3+master.039e4ea9]
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) [securejarhandler-2.1.4.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) [?:?]
	at me.lib720.caprica.vlcj.player.embedded.videosurface.ByteBufferFactory.getAddressOffset(ByteBufferFactory.java:67) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.embedded.videosurface.ByteBufferFactory.<clinit>(ByteBufferFactory.java:38) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.embedded.videosurface.NativeBuffers.allocate(NativeBuffers.java:74) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.embedded.videosurface.CallbackVideoSurface$SetupCallback.format(CallbackVideoSurface.java:93) [watermedia-1.2.0.jar%23366!/:1.2.0]
[21:25:09] [media-events/ERROR]: An error occurred building event handler
java.lang.NullPointerException: Cannot invoke "java.lang.ClassLoader.loadClass(String)" because the return value of "java.lang.Thread.getContextClassLoader()" is null
	at net.minecraftforge.eventbus.EventSubclassTransformer.buildEvents(EventSubclassTransformer.java:92) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventSubclassTransformer.transform(EventSubclassTransformer.java:44) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventBusEngine.processClass(EventBusEngine.java:26) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.service.ModLauncherService.processClassWithFlags(ModLauncherService.java:32) ~[eventbus-6.0.3.jar:6.0.3+6.0.3+master.039e4ea9]
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) [securejarhandler-2.1.4.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) [?:?]
	at me.srrapero720.watermedia.api.video.players.VideoLanPlayer$1.mediaPlayerReady(VideoLanPlayer.java:379) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerReadyEvent.notify(MediaPlayerReadyEvent.java:38) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerReadyEvent.notify(MediaPlayerReadyEvent.java:30) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager.raiseEvent(NativeEventManager.java:179) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.EventApi.raiseEvent(EventApi.java:83) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.MediaPlayerReadyEventHandler.positionChanged(MediaPlayerReadyEventHandler.java:59) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerPositionChangedEvent.notify(MediaPlayerPositionChangedEvent.java:41) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerPositionChangedEvent.notify(MediaPlayerPositionChangedEvent.java:30) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager.raiseEvent(NativeEventManager.java:179) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager$EventCallback.callback(NativeEventManager.java:206) [watermedia-1.2.0.jar%23366!/:1.2.0]
[21:25:09] [media-events/ERROR]: An error occurred building event handler
java.lang.NullPointerException: Cannot invoke "java.lang.ClassLoader.loadClass(String)" because the return value of "java.lang.Thread.getContextClassLoader()" is null
	at net.minecraftforge.eventbus.EventSubclassTransformer.buildEvents(EventSubclassTransformer.java:92) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventSubclassTransformer.transform(EventSubclassTransformer.java:44) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.EventBusEngine.processClass(EventBusEngine.java:26) ~[eventbus-6.0.3.jar:?]
	at net.minecraftforge.eventbus.service.ModLauncherService.processClassWithFlags(ModLauncherService.java:32) ~[eventbus-6.0.3.jar:6.0.3+6.0.3+master.039e4ea9]
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?]
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) [securejarhandler-2.1.4.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) [securejarhandler-2.1.4.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) [?:?]
	at me.srrapero720.watermedia.api.video.players.VideoLanPlayer$1.paused(VideoLanPlayer.java:269) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerPausedEvent.notify(MediaPlayerPausedEvent.java:36) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerPausedEvent.notify(MediaPlayerPausedEvent.java:28) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager.raiseEvent(NativeEventManager.java:179) [watermedia-1.2.0.jar%23366!/:1.2.0]
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager$EventCallback.callback(NativeEventManager.java:206) [watermedia-1.2.0.jar%23366!/:1.2.0]
```

**Comment by MeixDev** [comment]
Can confirm, with the latest version of WaterFrames for 1.19.2 and last version of WaterMedia

```
[01:48:37] [media-events/ERROR] [ne.mi.ev.EventSubclassTransformer/EVENTBUS]: An error occurred building event handler
java.lang.NullPointerException: Cannot invoke "java.lang.ClassLoader.loadClass(String)" because the return value of "java.lang.Thread.getContextClassLoader()" is null
	at net.minecraftforge.eventbus.EventSubclassTransformer.buildEvents(EventSubclassTransformer.java:92) ~[eventbus-6.0.3.jar:?] {}
	at net.minecraftforge.eventbus.EventSubclassTransformer.transform(EventSubclassTransformer.java:44) ~[eventbus-6.0.3.jar:?] {}
	at net.minecraftforge.eventbus.EventBusEngine.processClass(EventBusEngine.java:26) ~[eventbus-6.0.3.jar:?] {}
	at net.minecraftforge.eventbus.service.ModLauncherService.processClassWithFlags(ModLauncherService.java:32) ~[eventbus-6.0.3.jar:6.0.3+6.0.3+master.039e4ea9] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) [securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) [securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) [securejarhandler-2.1.4.jar:?] {}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) [?:?] {}
	at me.srrapero720.watermedia.api.video.players.VideoLanPlayer$1.opening(VideoLanPlayer.java:249) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerOpeningEvent.notify(MediaPlayerOpeningEvent.java:36) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerOpeningEvent.notify(MediaPlayerOpeningEvent.java:28) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager.raiseEvent(NativeEventManager.java:179) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager$EventCallback.callback(NativeEventManager.java:206) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
[01:48:37] [media-events/ERROR] [ne.mi.ev.EventSubclassTransformer/EVENTBUS]: An error occurred building event handler
java.lang.NullPointerException: Cannot invoke "java.lang.ClassLoader.loadClass(String)" because the return value of "java.lang.Thread.getContextClassLoader()" is null
	at net.minecraftforge.eventbus.EventSubclassTransformer.buildEvents(EventSubclassTransformer.java:92) ~[eventbus-6.0.3.jar:?] {}
	at net.minecraftforge.eventbus.EventSubclassTransformer.transform(EventSubclassTransformer.java:44) ~[eventbus-6.0.3.jar:?] {}
	at net.minecraftforge.eventbus.EventBusEngine.processClass(EventBusEngine.java:26) ~[eventbus-6.0.3.jar:?] {}
	at net.minecraftforge.eventbus.service.ModLauncherService.processClassWithFlags(ModLauncherService.java:32) ~[eventbus-6.0.3.jar:6.0.3+6.0.3+master.039e4ea9] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) [securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) [securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) [securejarhandler-2.1.4.jar:?] {}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) [?:?] {}
	at me.srrapero720.watermedia.api.video.players.VideoLanPlayer$1.buffering(VideoLanPlayer.java:254) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerBufferingEvent.notify(MediaPlayerBufferingEvent.java:41) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerBufferingEvent.notify(MediaPlayerBufferingEvent.java:30) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager.raiseEvent(NativeEventManager.java:179) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager$EventCallback.callback(NativeEventManager.java:206) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
[01:48:37] [media-events/ERROR] [ne.mi.ev.EventSubclassTransformer/EVENTBUS]: An error occurred building event handler
java.lang.NullPointerException: Cannot invoke "java.lang.ClassLoader.loadClass(String)" because the return value of "java.lang.Thread.getContextClassLoader()" is null
	at net.minecraftforge.eventbus.EventSubclassTransformer.buildEvents(EventSubclassTransformer.java:92) ~[eventbus-6.0.3.jar:?] {}
	at net.minecraftforge.eventbus.EventSubclassTransformer.transform(EventSubclassTransformer.java:44) ~[eventbus-6.0.3.jar:?] {}
	at net.minecraftforge.eventbus.EventBusEngine.processClass(EventBusEngine.java:26) ~[eventbus-6.0.3.jar:?] {}
	at net.minecraftforge.eventbus.service.ModLauncherService.processClassWithFlags(ModLauncherService.java:32) ~[eventbus-6.0.3.jar:6.0.3+6.0.3+master.039e4ea9] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) [securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) [securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) [securejarhandler-2.1.4.jar:?] {}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) [?:?] {}
	at me.srrapero720.watermedia.api.video.players.VideoLanPlayer$1.error(VideoLanPlayer.java:373) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerEncounteredErrorEvent.notify(MediaPlayerEncounteredErrorEvent.java:36) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.player.base.events.MediaPlayerEncounteredErrorEvent.notify(MediaPlayerEncounteredErrorEvent.java:28) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager.raiseEvent(NativeEventManager.java:179) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
	at me.lib720.caprica.vlcj.support.eventmanager.NativeEventManager$EventCallback.callback(NativeEventManager.java:206) [watermedia-1.2.0.jar%231076!/:1.2.0] {re:classloading}
```

The log ends with the following few lines:
```
Assertion failed!
Program: C:\Program Files\Eclipse Adoptium\jdk-17.0.4.101-hotspot\bin\javaw.exe
File: /builds/videolan/vlc/extras/package/win32/../../../lib/media_list_player.c, Line 531
Expression: p_mlp->i_refcount == 0
Process exited with code 3.
```

**Comment by SrRapero720** [comment]
fixed (WORKARROUND) on WATERMeDIA 1.2.10
update it: https://www.curseforge.com/minecraft/mc-mods/watermedia/files/4596094
